app/grandchallenge/publications/models.py

```diff
@@ -147,3 +147,11 @@ def ama_html(self):
         citation = re.sub(r"^1\. ", "", citation)
 
         return clean(citation)
+
+    @property
+    def authors(self):
```

Comment:
This might be quite slow, especially with how it is used in the filter. What we could do instead is extract this information out on import and then save it on the model. We can deploy it and see.